.github/workflows/pypi_upload.yml

```diff
@@ -17,12 +17,11 @@ jobs:
 
     - name: Install latest pip, setuptools, twine, wheel
       run: |
-        python -m pip install --upgrade pip setuptools twine wheel
+        python -m pip install --upgrade build pip setuptools twine wheel
```

Comment:
Do you still need setuptools and wheel here? Those should be pulled in in the isolated build environment, and if everything is working well the versions in the environment where `build ` is installed won't even be used.

Reply:
Sweet, thanks. Done.